proj/src/main/webapp/script.js

```diff
@@ -38,28 +35,34 @@ function appendMessage(payload){
 /*
     Realtime Database
  */
+
 const path = '/messages'; // can make this more detailed (for example add user ID)
+const LIMIT = 20; // how many messages to load at a time
+var FIRST_CHILD_KEY;
```

Comment:
I think it is good that LIMIT is in all caps because it is a constant, but I think FIRST_CHILD_KEY isn't a constant since it gets set and mutated by other places, so maybe that one shouldn't be all caps.

Reply:
fixed!